demo/docker-compose.yaml

```diff
@@ -1,5 +1,81 @@
 version: "3.9"
 services:
+  starrocks:
```

Comment:
We're up to a pretty large number of containers at this point. Does it make sense to have all the engines running at once or can we use something like docker compose profiles so people can run the demo with just some subset of the containers? https://docs.docker.com/compose/profiles/

Reply:
I agree.   don't commit this to main.